bin/oneoff/levelbuilder_permission

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env ruby
+#
+# This script adds a row to the user_permissions table with permission type
+# 'levelbuilder' for all admin users. The intent is for this script to be run
+# in the levelbuilder environment so that we can break the connection between
+# levelbuilder access and admin privileges.
+#
+# This is being done as a script rather than a migration as it only needs to
+# (should) happen in the levelbuilder environment.
+
+require_relative '../../dashboard/config/environment'
+require src_dir 'database'
+
+time_now = DateTime.now
+User.where(admin: true).find_each do |user|
```

Comment:
Note that this (intentionally) does not grant the `levelbuilder` permission to deleted accounts that were admins.